src/Plugin/migrate/process/ValidEDTF.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Drupal\migrate_islandora_csv\Plugin\migrate\process;
+
+use Drupal\Component\Plugin\ConfigurableInterface;
+use Drupal\controlled_access_terms\EDTFUtils;
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\MigrateSkipRowException;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+
+/**
+ * Skips processing the current row when the EDTF date isn't valid.
+ *
+ * Available configuration keys:
+ * - intervals (optional): Boolean of whether this field is supporting intervals
+ *   or not, defaults to TRUE.
+ * - sets (optional): Boolean of whether this field is supporting sets or not,
+ *   defaults to TRUE.
```

Comment:
Sets is suffering the same issue as intervals did. The default appears to be FALSE instead of TRUE, even after applying the most recent commit.

Reply:
Hey @seth-shaw-unlv when dumping out the configuration coming in at https://github.com/Islandora/migrate_islandora_csv/pull/41/files#diff-1326bc15007284c4eea1331864e61d9549eadd82c97d579fe4af181ac58b053aR40 I see:

```
Array
(
    [plugin] =&gt; valid_edtf
    [intervals] =&gt; 1
    [sets] =&gt; 1
    [strict] =&gt; 
)
```

Where my migrate YML is:

```
field_edtf_date_created:
    - plugin: skip_on_empty
      method: process
      source: 'Date_Created'
    - plugin: valid_edtf
```

Reply:
Huh, when I slipped a `print_r($this->configuration,TRUE)` into the transform function it showed me:
```
 Array
(
    [plugin] => valid_edtf
    [source] => date
)
```

So the default configs aren't getting set.

Reply:
I just slipped additional print statements into the other functions and neither defaultConfiguration nor getConfiguration were ever called. Only setConfiguration was.

Reply:
I think this is because, even though this class is implementing ConfigurableInterface, that ProcessPluginBase doesn't extend any classes that use the defaultConfiguration or getConfiguration functions making them moot.

Reply:
Absolute 🤦  moment. We have this plugin living in client code and moving it over here instead of cherrypicking I have the get/set backwards. Upped a playbook box with this plugin and modified the yml to use it and saw the same behavior. Commit forthcoming, apologies for wasting your time!